docker-for-windows/index.md

```diff
@@ -50,6 +50,8 @@ The next few steps take you through some examples. These are just suggestions fo
 
 2.  Run some Docker commands, such as `docker ps`, `docker version`, and `docker info`.
 
+	**Note for Windows Users**: Remember to use double quotes (") instead of single quotes (') when specifying parameters in docker commands while using CMD/Powershell.
```

Comment:
This is a useful note. I'd like to see it moved to a location preceding this numbered list.